test/functional_test.js

```diff
@@ -18,31 +15,17 @@ const responses = {};
 
 // Expects header names to be lowercase in this object.
 const expectedHeaders = {
-    'accept-ranges': 'bytes',
     'access-control-allow-origin': '*',
-    'cache-control': 'public, max-age=31536000',
-    'connection': 'Keep-Alive',
-    'content-length': '',
+    'cache-control': 'public, max-age=31919000',
```

Comment:
How is this number computed? The previous one was 1 year.

Reply:
Not sure, its set by the CDN when choosing "1 year" as caching time in the UI